src/items/FileItem.tsx

```diff
@@ -136,6 +137,23 @@ const FileItem = ({
             pdfViewerLink={pdfViewerLink}
           />
         );
+      } else if (
+        MimeTypes.isDocument(mimetype) ||
+        MimeTypes.isPresentation(mimetype) ||
+        MimeTypes.isSpreadsheet(mimetype)
+      ) {
+        return (
+          <Box height='100%' width='100%' id={id}>
+            <FileDocument uri={url} />
+            <DownloadButtonFileItem
+              id={id}
+              name={originalFileName ?? item.name}
```

Comment:
I think we want to show the `item.name`
```suggestion
              name={item.name}
```